nixos/lib/test-driver/pyproject.toml

```diff
@@ -35,11 +35,6 @@ ignore_missing_imports = true
 module = "junit_xml.*"
 ignore_missing_imports = true
 
-[tool.black]
-line-length = 88
-target-version = ['py39']
```

Comment:
`ruff` defaults to 3.8 until v0.8.0 which defaults to 3.9: should we perhaps be explicit about which version to use?

Reply:
Now targeting python 3.12 (our default version).